deploy/crds/apps.m88i.io_v1alpha1_nexus_cr.yaml

```diff
@@ -20,9 +20,9 @@ spec:
   # Data persistence details
   persistence:
     # Should we persist Nexus data? Yes, please. (turn this to false only if you're evaluating this resource)
-    persistent: true
+    persistent: false
```

Comment:
Should this be set to false here? If so, we might want to change the comment above it (specifically the "Yes, please" part).